package.json

```diff
@@ -11,7 +11,7 @@
     "clean": "shx rm -rf ./{coverage,dist,docs,node_modules,tmp}/ ./{packages,e2e}/*/{dist,node_modules}/ && git clean -fX -e \"!.env*,nx-cloud.env\"",
     "ci:release": "pnpm publish -r --no-git-checks && changeset tag",
     "ci:version": "changeset version && pnpm install --no-frozen-lockfile",
-    "changeset": "pnpm changeset",
```

Comment:
This causes infinite recursion of the command so it was my mistake when I added it. We don't "need" this command, its the same thing as calling `pnpm changeset` without it here, but it at least makes the command discoverable for those not aware.

`pnpm changeset` is the same thing as `pnpm changeset add`